test/test_ordered_set.jl

```diff
@@ -136,9 +136,9 @@
 
     # find
     s = OrderedSet([1,3,5,7])
-    @test findfirst(s,1) == 1
-    @test findfirst(s,7) == 4
-    @test findfirst(s,2) == 0
+    @test findfirst(equalto(1), s) == 1
+    @test findfirst(equalto(7), s) == 4
+    @test findfirst(equalto(2), s) == 0
```

Comment:
I guess these are coming in from `Compat` ?
It is a nice feature

Reply:
Yes, https://github.com/JuliaLang/julia/pull/23812 on master and https://github.com/JuliaLang/Compat.jl/pull/405 in Compat.